GETTING_STARTED.md

```diff
@@ -68,70 +77,140 @@ Elasticsearch will need to be on [version 5](https://www.elastic.co/guide/en/ela
+  * use `$ make debug DATASET_ROUTES_ENABLED=true`
+* [florence](https://github.com/ONSdigital/florence)
+  * use `$ make debug ENABLE_DATASET_IMPORT=true ENCRYPTION_DISABLED=true`
+*
 
 If you already have content, and you just want to run the web journey, you'll need the [dataset](#dataset-journey), [filter](#filter-journey) and [web](#web) services.
 
 #### Dataset journey:
 * [dp-dataset-api](https://github.com/ONSdigital/dp-dataset-api)
+  * use `$ make debug`
 * [dp-frontend-dataset-controller](https://github.com/ONSdigital/dp-frontend-dataset-controller)
+  * use `$ make debug`
 * [dp-frontend-filter-dataset-controller](https://github.com/ONSdigital/dp-frontend-filter-dataset-controller)
+  * use `$ make debug`
 
 #### Import services:
 * [dp-recipe-api](https://github.com/ONSdigital/dp-recipe-api)
+  * use `$ make debug`
 * [dp-import-api](https://github.com/ONSdigital/dp-import-api)
+  * use `$ make debug`
 * [dp-import-tracker](https://github.com/ONSdigital/dp-import-tracker)
+  * use `$ make debug`
 * [dp-dimension-extractor](https://github.com/ONSdigital/dp-dimension-extractor)
+  * use `$ make debug ENCRYPTION_DISABLED=true`
 * [dp-dimension-importer](https://github.com/ONSdigital/dp-dimension-importer)
+  * use `$ make debug`
 * [dp-observation-extractor](https://github.com/ONSdigital/dp-observation-extractor)
+  * use `$ make debug ENCRYPTION_DISABLED=true`
 * [dp-observation-importer](https://github.com/ONSdigital/dp-observation-importer)
+  * use `$ make debug`
 * [dp-hierarchy-builder](https://github.com/ONSdigital/dp-hierarchy-builder)
+  * use `$ make debug`
 * [dp-search-builder](https://github.com/ONSdigital/dp-search-builder)
+  * use `$ make debug`
 
 #### Filter journey:
 * [dp-search-api](https://github.com/ONSdigital/dp-search-api)
+  * use `$ make debug`
 * [dp-code-list-api](https://github.com/ONSdigital/dp-code-list-api)
+  * use `$ make debug`
 * [dp-hierarchy-api](https://github.com/ONSdigital/dp-hierarchy-api)
+  * use `$ make debug`
 * [dp-filter-api](https://github.com/ONSdigital/dp-filter-api)
+  * use `$ make debug`
 * [dp-dataset-exporter](https://github.com/ONSdigital/dp-dataset-exporter)
+  * use `$ make debug`
 * [dp-dataset-exporter-xlsx](https://github.com/ONSdigital/dp-dataset-exporter-xlsx)
+  * use `$ ./run.sh`
 * [dp-download-service](https://github.com/ONSdigital/dp-download-service)
+  * use `$ make debug`
+
+[dp-api-router](https://github.com/ONSdigital/dp-api-router) locally as well.
+  * use `$ make debug`
 
-Make sure other apps are on CMD branches, optionally try and run the [dp-api-router](https://github.com/ONSdigital/dp-api-router) locally as well.
 
 The majority of these apps will run with `make debug`, except the XLSX exporter which requires `./run.sh`
+
+### iTerm2 setup
+  - Any terminal can be used however there are advantages of using iTerm2 such as badges, profiles, arrangements, panes ability to send commands to all panes.
```

Comment:
I'm not sure `panes ability to send commands to all panes` makes sense? at the least there should probably be an `and` before it

Reply:
Ah cheers missing comma :D